test/test_helper.rb

```diff
@@ -1,7 +1,7 @@
 require 'bundler/setup'
 
 require 'active_fulfillment'
-
+require 'rexml/document'
```

Comment:
Ahh so we use rexml in the tests - bummer. In that case I like your approach of splitting this up. As @garethson suggested can we split this by provider so we can ship smaller changes on our side? Then as a last step lets updates the specs.